docs/user_guide/accessibility.md

```diff
@@ -80,71 +93,71 @@ For a simple site (no extra languages or versions), ensure `sphinx-sitemap`
 is installed in your documentation environment, and modify your `conf.py`:
 
 ```python
-    extensions += ["sphinx_sitemap"]
+extensions += ["sphinx_sitemap"]
 
-    html_baseurl = os.environ.get("SPHINX_HTML_BASE_URL", "http://127.0.0.1:8000/")
-    sitemap_locales = [None]
-    sitemap_url_scheme = "{link}"
+html_baseurl = os.environ.get("SPHINX_HTML_BASE_URL", "http://127.0.0.1:8000/")
+sitemap_locales = [None]
+sitemap_url_scheme = "{link}"
 ```
 
 :::
 
 ### Logo best practices
 
-If you use both light and dark themes, it's best to provide a logo that works well in both or to provide an alternative for the dark theme.
-If you have a logo, you can add alt-text to it by adding the following to your
-`conf.py`:
+If you use both light and dark themes, it's best to provide a logo that works
+well in both or to provide an alternative for the dark theme. If you have a
+logo, you can add alt-text to it by adding the following to your `conf.py`:
 
 ```python
-  "logo": {
-      "text": "PyData Theme",
-      "image_dark": "_static/logo-dark.svg",
-      "alt_text": "PyData Theme home",
-  },
+"logo": {
+    "image_dark": "_static/logo-dark.svg",
+    "alt_text": "<Your Project> home",
+},
 ```
 
-Note the use of "home" in the alt text to indicate that the logo is also a link to the home page.
+Note the use of "home" in the alt text to indicate that the logo is also a link
+to the home page.
 
-### In the Browser
+### Test and inspect your site
 
-Several in-browser tools exist for interactively debugging the accessibility
-of a single page at a time and can be useful during the content development cycle.
+Several browser tools exist for interactively debugging the accessibility
+of a single page at a time and can be useful during the content development
+cycle.
 
-### Built-in tools
+Non-interactive tools also exist for use with CI (continuous integration).
 
-Most major browsers, including [Firefox](https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector)
-and [Chrome](https://developers.google.com/web/tools/chrome-devtools/accessibility/reference),
-have accessibility tools built-in as part of their web developer tools.
-These tools can help to quickly identify accessibility issues and often include links to standards.
+#### Browser tools
```

Comment:
Yeah, I had the same thought on my first pass but didn't add it because the document was structured a bit differently and I wasn't sure if we should promote a paid product. 

But given that we both find it useful, I agree with you that it could help some people out to know about it, so I added a sentence about it.